zh/enable-admission-webhook.md

```diff
@@ -21,6 +21,18 @@ TiDB Operator 在默认安装情况下不会开启准入控制器，你需要手
       create: true
     ```
 
+    TiDB Operator 准入控制器与大部分 Kubernetes 平台上产品的准入控制器较为不同，TiDB Operator 通过 [扩展 API-Server](https://kubernetes.io/docs/tasks/access-kubernetes-api/setup-extension-api-server/) + [WebhookConfiguration](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#configure-admission-webhooks-on-the-fly) 的配置组合。默认情况下，如果你的 Kubernetes 集群版本大于等于 1.13.0，你可以通过上述配置直接开启 Webhook 功能。
```

Comment:
```
TiDB Operator 通过[扩展 API-Server] + [WebhookConfiguration] 的配置组合。
```
This sentence is not smooth enough.

Reply:
Please add some explanation about why this webhook is different from others and why we extend the api-server + webhookconfig.

Reply:
It would be much complex to explain why we use api-server + webhookconfig here in a single document. I tend to show the configurations and steps for webhook in this document. We would leave the architecture and mechanism of the webhook in another document.

Reply:
Furthermore, this document is for the end-user operation, I think it is not necessary to explain the difference and reason here.